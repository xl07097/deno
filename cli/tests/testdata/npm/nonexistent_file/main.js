import hmacSHA512 from "npm:crypto-js/hmac-sha512";
