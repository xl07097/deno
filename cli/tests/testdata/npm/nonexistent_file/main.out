error: Unable to load [WILDCARD]hmac-sha512 imported from [WILDCARD]/testdata/npm/nonexistent_file/main.js

Caused by:
[WILDCARD]
